Formula/notmuch.rb

```diff
@@ -28,14 +27,12 @@ def install
     args = %W[
       --prefix=#{prefix}
       --mandir=#{man}
-      --with-emacs
       --emacslispdir=#{elisp}
       --emacsetcdir=#{elisp}
       --without-ruby
     ]
 
-    # Emacs and parallel builds aren't friends
-    ENV.deparallelize
+    ENV.append_path "PYTHONPATH", Formula["sphinx-doc"].opt_libexec/"lib/python3.7/site-packages"
```

Comment:
This needs to be 3.8 then

Reply:
`sphinx-doc` doesn't have a `python3.8` path:

```console
$ ls /usr/local/opt/sphinx-doc/libexec/lib/
python3.7
```

This is with the most up-to-date version of `sphinx-doc` from `master` as of 2020-02-21.

Reply:
Oh, got my paths mixed up. Carry on